### PR TITLE
some refactoring

### PR DESCRIPTION
**Pull request by fperrad**

### CODE REVIEW

**Comment by minad** [comment]
I don't think this is more readable :-1:

File: mp_div_2d.c [hunk 1]
```diff
@@ -34,16 +34,15 @@ mp_err mp_div_2d(const mp_int *a, int b, mp_int *c, mp_int *d)
...
+      mp_digit mask = ((mp_digit)1 << b) - 1uL;
 
       /* shift for lsb */
-      shift = (mp_digit)(MP_DIGIT_BIT - b);
+      mp_digit shift = (mp_digit)(MP_DIGIT_BIT - b);
 
       /* carry */
-      r = 0;
+      mp_digit r = 0;
       for (x = c->used; x --> 0;) {
```
Comment (minad) [inline mp_div_2d.c]: This is good however.

Comment (sjaeckel) [reply mp_div_2d.c]: but would break c89

Comment (minad) [reply mp_div_2d.c]: No these are still only a list of inits. We check c89 with a compiler flag.

**Comment by minad** [comment]
Close this since it is outdated. And we probably don't want these single line initializations.

**Pull request closed without merging**